docs/source/whatsnew/1.0.0rc1.rst

```diff
@@ -57,6 +57,14 @@ API Changes
   to generate reference probabilistic persistence forecasts (:pull:`490`)
 * :py:func:`solarforecastarbiter.reference_forecasts.utils.get_data_start_end`
   now requires an `issue_time` argument (:pull:`497`)
+  * Added a ``missing_forecast`` parameter to
```

Comment:
```suggestion
  * Added a ``forecast_fill_method`` parameter to
```

Reply:
that's what i get for trying to rush. sorry about these